common/patches/rof.cpp

```diff
@@ -34,6 +34,7 @@
 #include "../rulesys.h"
 #include "../path_manager.h"
 #include "../races.h"
+#include "../../zone/raids.h"
```

Comment:
common shouldn't be including anything from `world` or `zone`

Reply:
Suggest moving the types you're referencing to `common` so that `zone` and `world` reference it instead of the other way around